lib/saluki-event/src/eventd/mod.rs

```diff
@@ -89,10 +119,18 @@ impl Priority {
             _ => Some(Priority::Normal),
         }
     }
+
+    /// Returns stringified name of the Priority variant.
+    pub fn stringify(self) -> &'static str {
```

Comment:
This should be `as_str`.

Reply:
I took out the doc since the `fn` doesn't actually need to be `pub` but lmk if you want it back in lol